gravitee-apim-console-webui/src/portal/navigation-items/section-editor-dialog/section-editor-dialog.component.ts

```diff
@@ -72,15 +73,22 @@ export class SectionEditorDialogComponent implements OnInit {
 
   private readonly dialogRef = inject(MatDialogRef<SectionEditorDialogComponent, SectionEditorDialogResult>);
   private readonly data: SectionEditorDialogData = inject(MAT_DIALOG_DATA);
+  public buttonTitle: string;
 
   constructor() {
     this.type = this.data.type;
     this.mode = this.data.mode;
-    this.title = `Add ${this.type.toLowerCase()}`;
+    if (this.mode === 'create') {
+      this.title = `Add ${this.type.toLowerCase()}`;
+    } else {
+      this.title = `Edit "${this.data.existingItem.title}" ${this.type.toLowerCase()}`;
+    }
+    this.buttonTitle = this.mode === 'create' ? 'Add' : 'Save';
```

Comment:
How about merging the two conditions?

```suggestion
    if (this.mode === 'create') {
      this.title = `Add ${this.type.toLowerCase()}`;
      this.buttonTitle = 'Add';
    } else {
      this.title = `Edit "${this.data.existingItem.title}" ${this.type.toLowerCase()}`;
      this.buttonTitle = 'Save';
    }